Sources/MapboxMaps/Style/Style.swift

```diff
@@ -433,7 +410,6 @@ extension Style: StyleManagerProtocol {
 
     // MARK: Layer Properties
 
-    /// :nodoc:
     public func _layerProperty(for layerId: String, property: String) -> StylePropertyValue {
```

Comment:
if we are removing nodoc, can we add docs to this since it is a public API

Reply:
The docs are inherited from the StyleManagerProtocol, which already had `:nodoc:` - this was about removing the redundancy.